### DISCUSSION TITLE
Add `disableTransitionExclude` config

### DISCUSSION THREAD

**Discussion by mancuoj** [abereghici/remix-themes#35] [Ideas]
@abereghici Thanks for developing Remix Themes — super helpful! Because my theme switching button needs to use transition, I need to be able to whitelist some components. This can be achieved by using the `:not` pseudo-class of css, just like this:

```ts
document.createTextNode(
    `
	*${disableTransitionExclude.map(s => `:not(${s})`).join('')} {
	  -webkit-transition: none !important;
	  -moz-transition: none !important;
	  -o-transition: none !important;
	  -ms-transition: none !important;
	  transition: none !important;
	}
    `,
),
```

**Comment by abereghici** [comment]
Thanks @mancuoj . I created an issue from this discussion and I'll investigate it.
